webui/react/src/pages/ExperimentList.tsx

```diff
@@ -727,6 +731,19 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
         );
         ExperimentMoveModal.open();
       }
+      if (action === Action.RetainLogs) {
+        if (!settings?.row?.length && settings.row.length === 0) return;
+        setBatchRetainLogsExperimentIds(
+          settings.row.filter(
+            (id) =>
+              canActionExperiment(Action.RetainLogs, experimentMap[id]) &&
+              permissions.canModifyExperiment({
+                workspace: { id: experimentMap[id].workspaceId },
+              }),
+          ),
+        );
+        ExperimentRetainLogsModal.open();
```

Comment:
this modal not working in the old experiment table

<img width="681" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/a502aebe-bb3e-4483-8e50-699f1727e714">

Reply:
Fixed!